components/library/libzip/manifests/sample-manifest.p5m

```diff
@@ -0,0 +1,161 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/zipcmp
+file path=usr/bin/zipmerge
+file path=usr/bin/ziptool
+file path=usr/include/zip.h
+file path=usr/include/zipconf.h
+file path=usr/lib/$(MACH64)/cmake/libzip/libzip-config-version.cmake
+file path=usr/lib/$(MACH64)/cmake/libzip/libzip-config.cmake
+file path=usr/lib/$(MACH64)/cmake/libzip/libzip-targets-noconfig.cmake
+file path=usr/lib/$(MACH64)/cmake/libzip/libzip-targets.cmake
+link path=usr/lib/$(MACH64)/libzip.so target=libzip.so.5
+link path=usr/lib/$(MACH64)/libzip.so.5 target=libzip.so.5.4
+file path=usr/lib/$(MACH64)/libzip.so.5.4
+file path=usr/lib/$(MACH64)/pkgconfig/libzip.pc
+file path=usr/share/man/man1/zipcmp.1
```

Comment:
The sample-manifest needs an update. When I run gmake sample-manifest here it will contain the .cmake files.